src/nft/events.ts

```diff
@@ -10,12 +10,15 @@ export enum NFTEventName {
   NFT_BUY_BAG_REFUNDED = 'NFT Buy Bag Refunded',
   NFT_BUY_BAG_SIGNED = 'NFT Buy Bag Signed',
   NFT_BUY_BAG_SUCCEEDED = 'NFT Buy Bag Succeeded',
+  NFT_BUY_BAG_SUCCEEDED_WITH_ERC20 = 'NFT Buy Bag Succeeded With ERC20',
   NFT_FILTER_OPENED = 'NFT Collection Filter Opened',
   NFT_FILTER_SELECTED = 'NFT Filter Selected',
   NFT_LISTING_COMPLETED = 'NFT Listing Success',
   NFT_LISTING_SIGNED = 'NFT Listing Signed',
   NFT_PROFILE_PAGE_START_SELL = 'NFT Profile Page Start Sell',
   NFT_SELL_ITEM_ADDED = 'NFT Sell Item Added',
   NFT_SELL_START_LISTING = 'NFT List Page Start Listing',
+  NFT_TOKEN_SELECTED = 'NFT Token Selected',
+  NFT_TOKEN_SELECTOR_CLICKED = 'NFT Token Selector Clicked',
```

Comment:
This is related to buying correct? Maybe prefix with `NFT_BUY_`? At first, I read these as "selecting an NFT" since NFT's are also technically tokens